website/source/docs/acl/acl-auth-methods.html.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "docs"
+page_title: "ACL Auth Methods"
+sidebar_current: "docs-acl-auth-methods"
+description: |-
+  An Auth Method is a component in Consul that performs authentication against a trusted external party to authorize the creation of an appropriately scoped ACL token usable within the local datacenter.
```

Comment:
kait: I made some earlier comment about "Auth Method" vs "auth method". We should be consistent.